keyboards/wolf/frogpad/info.json

```diff
@@ -0,0 +1,63 @@
+{
+    "manufacturer": "Geon",
+    "keyboard_name": "Frog Pad",
+    "maintainer": "ToastyStoemp",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["F5", "F6", "B7", "B3"],
+        "rows": ["F0", "B6", "D6", "B5", "D7", "B4"]
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "F4", "pin_b": "B1" },
+            { "pin_a": "F1", "pin_b": "B2" }
+        ]
+    },
+    "processor": "atmega32u4",
+    "url": "https://geon.works/products/frog-pad",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0004",
+        "vid": "0x5453"
+    },
+    "layouts": {
+        "LAYOUT_orhto_6x4": {
```

Comment:
Is there a reason [`numpad_6x4`](https://github.com/qmk/qmk_firmware/tree/master/layouts/default/numpad_6x4) shouldn't be used? That would make more sense based off product photos.